jwm.1.in

```diff
@@ -1074,14 +1074,26 @@ that the value is relative to the bottom of the screen.
 .B "WINDOW STYLE"
 .RS
 The \fBWindowStyle\fP tag controls the look of window borders.
-This tag supports the following attribute:
+This tag supports the following attributes:
 .P
 .B decorations
 .RS
 The window decorations to use. Valid options are \fBflat\fP and
 \fBmotif\fP. \fBflat\fP is the default.
 .RE
 .P
+.B machinename
```

Comment:
I'm kind of conflicted on the name. As is it sounds like you're setting the machine name. Even the code has showMachineName, but that's a little on the long side. Should be named `showmachine` instead?

Reply:
actually, I think the attribute should be **showclient** since it is based on the **WM_CLIENT_MACHINE** property.